src/api/cmp/vertical.rs

```diff
@@ -11,21 +11,21 @@ macro_rules! impl_cmp_vertical {
             /// Lane-wise equality comparison.
             #[inline]
             pub fn eq(self, other: Self) -> $mask_ty {
-                use llvm::simd_eq;
+                use super::llvm::simd_eq;
```

Comment:
I'd prefer to refer to this with an absolute path if possible. Both are brittle, but I think if we ever move the `llvm` module the absolute path would be easier to fix.